account_constraints/__init__.py

```diff
@@ -18,3 +18,4 @@
 #
 ##############################################################################
```

Comment:
This is just a small bugfix PR, so I would think this might not be the place to make unchanged parts of the code standards compliant.